engine/runner/src/main/scala/org/enso/runner/DependencyPreinstaller.scala

```diff
@@ -28,8 +30,8 @@ object DependencyPreinstaller {
     * to find all transitive dependencies and ensures that all of them are
     * installed.
     */
-  def preinstallDependencies(projectRoot: File, logLevel: LogLevel): Unit = {
-    val logger = Logger[DependencyPreinstaller.type]
+  def preinstallDependencies(projectRoot: File, logLevel: Level): Unit = {
+    val logger = LoggerFactory.getLogger(classOf[DependencyPreinstaller.type])
```

Comment:
This seems like _a usage of logger_. Does that mean we are supposed to use `org.slf4j` throughout the code base to do some logging?